1-exercises/A-reading-list/script.js

```diff
@@ -1,27 +1,52 @@
 function readingList(books) {
-  // Write your code here...
+  const content = document.querySelector("#content"); //Select div by id
+
+  //loop to create each element of the list
+  for (let key in books) {
+    const paragraphList = document.createElement("p"); //create element paragraph
+    paragraphList.innerHTML = `${books[key].title} by ${books[key].author} `; //.innerHTML -> sets or returns the HTML content
+
+    const pictureList = document.createElement("img"); //create element img
+    pictureList.setAttribute("src", books[key].bookCoverImage); //pass att to pictureList(The attribute name, valuer[Href])
+    pictureList.style.width = "200px"; //Resize to width 200px
```

Comment:
This should be narrower to look like the example. You could use CSS to set the width. You can also use percentage rather than px, but either is OK.